cnxarchive/tests/test_database.py

```diff
@@ -1539,6 +1539,7 @@ def test_new_module_wo_uuid(self, cursor):
         self.assertEqual(uuid_, controls_uuid)
         self.assertEqual(license_id, controls_license_id)
 
+
```

Comment:
Could you remove the extra blank lines around methods?
There should only be one blank line between methods.

Running pep8 gives:
```
cnxarchive/tests/test_database.py:1543:5: E303 too many blank lines (2)
cnxarchive/tests/test_database.py:1602:5: E303 too many blank lines (2)
cnxarchive/tests/test_database.py:1642:5: E303 too many blank lines (2)
```

I know our code is not yet pep8 compliant, but we're trying to be. :)